src/components/stepper/index.js

```diff
@@ -4,7 +4,7 @@ import Button from "../button";
 import "./style.css";
 import { useState } from "react";
 
-const Stepper = ({ header, children, onComplete }) => {
+const Stepper = ({ data, header, children, onComplete }) => {
```

Comment:
is this a generic Stepper that can be reused or a specific one? if specific, it needs renaming to be specific, if generic, we should avoid the validate input because that is very specific